scripts/copy_output.lua

```diff
@@ -53,6 +54,11 @@ for _, image in ipairs(lib.images) do
 	end
 end
 
+local kernel_debug_source = string.format('openwrt/bin/targets/%s/kernel-debug.tar.bz2',
+	bindir)
+local kernel_debug_dest   = env.GLUON_IMAGEDIR..'/debug/'..env.BOARD..'-'..subtarget..'-kernel-debug.tar.bz2'
```

Comment:
Instead of `env.BOARD..'-'..subtarget`, using the Gluon target name - found in the variable *target* - would be more appropriate for a Gluon output file.

Reply:
Ok, I'll give that a try.